db/seeds.rb

```diff
@@ -17,7 +17,55 @@
   insurance: 'Yes',
   can_accomodate_meetings: true,
   areas_of_interest: ['Providing ongoing assistance to sponsors through SSP matching'],
-  comments: ''
+  comments: '',
+  private_notes: 'Private notes'
 })
-
 puts "Created lawyer #{lawyer.email} with password 'test'"
+
+student = Student.create!(name: "Example Student", email: "example_student@example.com", password: "test", approved: true, extra: {
+  approved: true,
+  telephone: '0016138289375',
+  city: 'Ottawa',
+  province: 'Ontario',
+  university: 'Carleton',
+  language: 'English',
+  course: 'CS1037',
+  campus_group: 'Carleton students',
+  private_notes: 'Private notes'
+})
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.